### ISSUE TITLE
--help and -h are slightly outdated. 

### ISSUE THREAD

**Issue by NomadicVoxel** [Genymobile/scrcpy#3122]
TLDR: builtin manual file is missing a few new notes. The ones I caught were those about the mouse's thumb buttons. 

 - [X] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [X] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Win11
 - scrcpy version: 1.22
 - installation method: Chocolatey
 - device model: Moto G Stylus
 - Android version: 11

**Describe the bug**
The help file is missing a few notes. 
- Mouse4 (thumb button, forward) opens the Notification Drawer.
- Mouse5 (thumb button, back) opens Switch Apps. 

While we're at it, it's kinda long and cluttered. Any chance we could break it into notes related to input/output, the display, and to recording?

**Comment by rom1v** [comment]
> The help file is missing a few notes.
>
>    Mouse4 (thumb button, forward) opens the Notification Drawer.
>    Mouse5 (thumb button, back) opens Switch Apps.

Yes, you could open a PR to fix it (changes must be done in app/src/cli.c).

> While we're at it, it's kinda long and cluttered. Any chance we could break it into notes related to input/output, the display, and to recording?

The README is more or less split into sections. The help and manpage are in alphabetical order, I'm not sure I want to add/maintain sections here.

**Comment by rom1v** [comment]
Fixed by 326897a0d4ab3ea0cf741cd87f01d2703db38374.
